### PR TITLE
Added a method to list the codes and values of terms of a taxonomy.

### PR DESCRIPTION
**Pull request by Daniel-KM**
Hi,

Added a method to list the codes and values of terms of a taxonomy.

Sincerely,

Daniel Berthereau
Infodoc & Knowledge management

### CODE REVIEW

**Comment by jajm** [comment]
The new method `listByTaxonomy` is not used anywhere. Are you using it in another plugin ?

File: views/admin/taxonomy-term/list.php [hunk 1]
```diff
@@ -1,5 +1,10 @@
-<?php echo head(array('title' => __('%s taxonomy terms', $taxonomy->name))); ?>
-<?php echo flash(); ?>
+<?php
+$total = count($terms);
+echo head(array(
+    'title' => __('%s taxonomy terms (%s)',
+        $taxonomy->name, empty($total) ? __('none') : ($total == 1 ? __('%d term', $total) : __('%d terms', $total))),
```
Comment (jajm) [inline views/admin/taxonomy-term/list.php]: Use __(plural('one term', '%s terms', $total)) instead. It improves translatability for languages where plural forms are handled differently

Comment (Daniel-KM) [reply views/admin/taxonomy-term/list.php]: plural() is hard to use with multiple specifiers in the main sentence and it doesn't manage the 'none' case. Anyway, I published a simplified title.

**Comment by Daniel-KM** [comment]
Hi,

I use the method in the request open_taxonomy, that should be merged after. It improves the speed of the creation of the form when there are a lot of taxonomies and terms.

Sincerely,

Daniel Berthereau
Infodoc & Knowledge management

**Pull request merged**